src/plugins/intel_gpu/tests/fusions/gemm_fusion_test.cpp

```diff
@@ -357,6 +360,9 @@ TEST_P(gemm_2in_act_scale_eltwise, broadcast_eltwise) {
         eltwise("sum", { "activation", "eltwise_data" }, eltwise_mode::sum,  data_types::f32),
         reorder("reorder_bfyx", "sum", p.default_format, data_types::f32)
     );
+    //Activation won't be fused because onednn doesn't support negative activation
+    if(engine.get_device_info().supports_immad)
+        p.expected_fused_primitives+=2;
```

Comment:
Why +2? Won't activation + eltwise be fused?

Reply:
Yes. Current behavior is isolating activation node to make it select cldnn impl.

Reply:
https://github.com/openvinotoolkit/openvino/pull/12055#discussion_r917840956 This comment is outdated. Current implementation is option1 (do not fuse & do not be fused).

Reply:
No activation-eltwise fusion is intended behavior. That is +2 is correct. #11969 enabled it but it caused performance regression in some reason, and it was reverted. Refer fuse_eltwise_f in prepare_primitive_fusing.cpp.